.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Description
+[What is wrong? What the expected and current behaviours are?]
+
+
+## Code to reproduce
+[Add indicative code example to reproduce the reported case]
+
+
+## Preliminary solution
+[What should be added/changed/removed to match the expected behaviour?]
+
+
+## References
+[Add links to the related articles, packages, reference examples]
+
+
+## Environment
```

Comment:
We do not _always_ need numpy/scipy versions. They're handy, but we'll have a good idea from how they installed if that's going to be part of the puzzle. Again, I really think we should keep this - which comes across as homework for every last PR - minimal.